build.gradle

```diff
@@ -47,3 +48,7 @@ allprojects {
         maven { url "https://jitpack.io" }
     }
 }
+
+subprojects {
```

Comment:
As a note, for the section above, should that apply to all projects or just subprojects? I'm using subprojects below since the relative gradle path matters